Source/Applications/Web/ApplicationForListingModel.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+import { Guid } from '@dolittle/rudiments';
+import { MicroserviceForListing } from './MicroserviceForListing';
+
+export type ApplicationForListingModel = {
```

Comment:
Just for consistency - should this be called `ApplicationForListing`? The other one is called `MicroserviceForListing` and the `...Model` suffix is usually used on the `...ViewModel` classes.

Reply:
Good catch - I missed that, I cannot remember my reasoning for suffixing with Model, but I will remove it.